### PR TITLE
Change page title to Tilup

### PR DESCRIPTION
**Pull request by MrPerux**
Previously the page title was 'Vite App'. This commit changes this to 'Tilup'

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -5,12 +5,12 @@
   <meta charset="UTF-8" />
   <link rel="icon" type="image/x-icon" href="/src/favicon.ico" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Vite App</title>
+  <title>Tilup</title>
```
Comment (DaClemens) [inline frontend/index.html:8]:
```suggestion
  <title>Tilup 🌷</title>
```

Comment (MrPerux) [reply frontend/index.html:8]: Good change. Now it looks ready to merge.

Comment (RageKnify) [reply frontend/index.html:8]: Should be Tulip instead of Tilup, no?

**Comment by RickdeJager** [comment]
Closing in favor of #3 ;)

**Pull request closed without merging**